packages/@aws-cdk/cdk/test/cloudformation/test.fn.ts

```diff
@@ -0,0 +1,11 @@
+import nodeunit = require('nodeunit');
+import fn = require('../../lib/cloudformation/fn');
+
+export = nodeunit.testCase({
+    'Fn::Join': {
+        'rejects empty list of arguments to join'(test: nodeunit.Test) {
```

Comment:
Are you sure there aren’t already tests for these?

Reply:
Reasonably:

1) Nothing showed up in `/test/` when I grep'd for `FnJoin(`.
2) There was no `test.fn.ts` and that's where I expect those unit tests to be